2025/ideas-list.md

```diff
@@ -9,6 +9,7 @@ page of each organization under the NumFOCUS umbrella at this page.
 - [ArviZ](https://github.com/arviz-devs/arviz/wiki/GsoC-2025-projects)
 - [conda / rattler](https://github.com/conda/rattler/issues/1058)
 - [Data Retriever](https://github.com/weecology/retriever/wiki/GSoC-2025-Project-Ideas)
+- [DISCOVER Cookbook](https://github.com/numfocus/DISCOVER-Cookbook/discussions/208)
```

Comment:
Google expects some level of organization for the project. Please take a look at the ideas submitted by your peers to refine your proposal accordingly. Thank you!

- [aeon](https://github.com/aeon-toolkit/aeon-admin/blob/main/gsoc/gsoc-2025-projects.md)
- [AiiDA](https://github.com/aiidateam/aiida-core/wiki/GSoC-2025-Projects)

Reply:
Okay I added the recommended structure. Let me know if there is anything further you would like to see before merging! Thanks.